.drone.yml

```diff
@@ -5,11 +5,14 @@ bootstrap_common: &bootstrap_common
     - mkdir -p /opt/metwork-mfcom-${DRONE_BRANCH}
     - ./bootstrap.sh /opt/metwork-mfcom-${DRONE_BRANCH} /opt/metwork-mfext-$${MFEXT_BRANCH}
     - cat adm/root.mk
+    - CACHE_HASH=`/opt/metwork-mfext-$${MFEXT_BRANCH}/bin/_build_cache_hash.sh .`
+    - if ! test -f /buildcache/build_mfcom_$${DRONE_BRANCH}_$${CACHE_HASH}; then echo $${CACHE_HASH} >.drone_cache; fi
```

Comment:
`${DRONE_BRANCH}` and not `$${DRONE_BRANCH}` as it is a DRONE env and not strictly a shell one

Reply:
Done